dashboards/Prometheus.json

```diff
@@ -3647,14 +3647,11 @@
                     "steppedLine": false,
                     "targets": [
                         {
-                            "expr": "time() - node_time_seconds",
-                            "format": "time_series",
+                            "expr": "clamp_min((time() - max by (node_name) (node_time_seconds)) - scalar(floor(prometheus_target_interval_length_seconds{quantile=\"0.99\",interval=~\"$scrapingInterval\"})),0)",
```

Comment:
Why `floor()` and not `round()`? And why round to integer seconds at all?

Reply:
Thank you. Round() is better. 

![Screenshot_20200122_132727](https://user-images.githubusercontent.com/25184101/72890842-7f74e500-3d1b-11ea-9d1f-b9faeaf4d267.png)